lib.php

```diff
@@ -1251,6 +1251,14 @@ function facetoface_get_userfields() {
                 $userfields[$shortname] = get_string($shortname);
             }
             $userfields['managersemail'] = get_string('manageremail', 'facetoface');
+
+            if ($DB->record_exists('user_info_field', array('shortname' => 'stream'))) {
```

Comment:
User profile field data (user_info_field) is mostly considered legacy data now for anything organisation related now, as it can all be sourced from the user's assignments.
This means that the implementation will be a little more complex as you can't use the existing Face-to-Face (F2F) custom field injection and will need to add it to the export data directly.

I also don't think we have a simple single function call to get something like the stream based on an assignment outside of querying the metadata tables, but I've been a bit weary on using them in code where we have the potential to resolve from live data. @matthewfein what do you think?